### ISSUE TITLE
v2.0.7: inconsistencies in `SedStyle`

### ISSUE THREAD

**Issue by agarny** [fbergmann/libSEDML#67]
`SedLine`-related things in `SedStyle` used to be named `xxxLine()`, but with the release of v2.0.7 they are now named `xxxLineStyle()`. I can see why (to avoid conflicts with `SedBase`'s `getLine()` method. However, two issues:
 - `SedStyle` defines `SedLine* mLine`, which shadows `SedBase::mLine`. So, I would expect `SedStyle::mLine` to be renamed to `SedStyle::mLineStyle`.
 - The renaming of `xxxLine()` to `xxxLineStyle()` makes `xxxMarker()` inconsistent. So, I would expect `xxxMarker()` to be renamed to `xxxMarkerStyle()` and `SedStyle::mMarker` to `SedStyle::mMarkerStyle`.

**Comment by fbergmann** [comment]
I'm also not happy about the renaming, and if you have a better name, then that would be fine. I also dislike the redundancy of the Style object having `getXXXStyle` methods. But sure, i can rename the remaining things as well for 2.0.8 if you prefer.

**Comment by agarny** [comment]
> I'm also not happy about the renaming, and if you have a better name, then that would be fine.

Agreed. Unfortunately, I can't think of a better name at this stage.

> I also dislike the redundancy of the Style object having `getXXXStyle` methods.

Heartily agreed with that one too.

> But sure, i can rename the remaining things as well for 2.0.8 if you prefer.

Yes, although I don't like the current names, I am also in favour of consistency... :)

**Comment by luciansmith** [comment]
I think we decided to stick with the current names--can this be closed?

**Comment by matthiaskoenig** [comment]
I am good with the way it is right now. Can be closed from my side.
